src/goto-programs/rebuild_goto_start_function.h

```diff
@@ -0,0 +1,28 @@
+/*******************************************************************\
+ Module: C Test Case Generator
+ Author: Thomas Kiley, thomas@diffblue.com
+\*******************************************************************/
+
+#ifndef CPROVER_GOTO_PROGRAMS_REBUILD_GOTO_START_FUNCTION_H
+#define CPROVER_GOTO_PROGRAMS_REBUILD_GOTO_START_FUNCTION_H
+
+#include <util/message.h>
+#include <util/symbol_table.h>
```

Comment:
Not much said, @peterschrammel said he would discuss this with you, but the gist of Peter's position is using includes everywhere it better than mixing includes when necessary with forward declarations.